doc/gpumd/input_parameters/compute_msd.rst

```diff
@@ -39,3 +55,8 @@ This means that you
 * the position data will be recorded every 5 steps
 * the maximum number of correlation steps is 200
 * you would like to compute only over group 1 in group method 1.
+
+
+To compute the :term:`MSD` for all groups in group method 1 and save a copy of the :term:`MSD` every 100 steps, one can write::
```

Comment:
Is 100 a good number? Isn't it too frequent? What a good value?

Reply:
I would say that a good number is probably around 100 000-1 000 000 steps. shorter simulations are probably so quick that it's not needed to save the msd intermittently

Reply:
then how about modify the number in the example?

Reply:
Will do!